denylist/src/denylist.rs

```diff
@@ -35,12 +32,13 @@ impl TryFrom<Vec<PublicKeyBinary>> for DenyList {
             tag_name: 0,
             client,
             filter,
+            valid_sign_keys: vec![],
         })
     }
 }
 
 impl DenyList {
-    pub fn new() -> Result<Self> {
+    pub fn new(settings: Settings) -> Result<Self> {
```

Comment:
Take `&Settings` so you don't have to clone settings when using this

Reply:
Fixed dbf8dd7